scripts/get-pr-feedback.sh

```
@@ -0,0 +1,133 @@
+
+if [[ "$UNRESOLVED_COUNT" -eq 0 ]]; then
+    printf "  ✓ No unresolved feedback\n\n"
+    exit 0
+fi
+
+# Process each unresolved thread
+echo "$UNRESOLVED_THREADS" | jq -c '.[]' | while read -r thread; do
+    THREAD_ID=$(echo "$thread" | jq -r '.id // ""')
+    PATH_FILE=$(echo "$thread" | jq -r '.path // "unknown"')
+    LINE=$(echo "$thread" | jq -r '.line // "?"')
+    AUTHOR=$(echo "$thread" | jq -r '.comments.nodes[0].author.login // "unknown"')
```

Shellcheck (0.11.0)

[warning] AUTHOR appears unused. Verify use (or export if used externally).

(SC2034)